### ISSUE TITLE
Pretrained weights for birds and comics give very low accuracy

### ISSUE THREAD

**Issue by HashmatShadab** [LiulietLee/CDTA#2]
Using the eval script with the pretrained target model weights, I am able to get a high clean accuracy for food dataset. But using the pretrained model weights for comics or bird's dataset gives a very low accuracy. Can you please check the weights again?

**Comment by LiulietLee** [comment]
Hi, I tested resnet34 and densenet161 on the BIRDS400 dataset and got an accuracy of about 96%. Did you use data normalization and set `model.eval()` during your testing?
